sw/device/tests/BUILD

```diff
@@ -2763,6 +2763,76 @@ opentitan_test(
     ],
 )
 
+opentitan_test(
+    name = "pwrmgr_normal_sleep_all_wake_ups",
+    srcs = ["pwrmgr_normal_sleep_all_wake_ups.c"],
+    cw310 = new_cw310_params(
+        test_cmd = """
+            --bootstrap={firmware}
+            --bitstream={bitstream}
```

Comment:
I believe that since #21645, you don't need to specify the bitstream anymore, @nbdd0121 ?